modules/media/jsx/mediaIndex.js

```diff
@@ -56,6 +54,63 @@ class MediaIndex extends React.Component {
     this.refs.mediaFilter.clearFilter();
   }
 
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {array} rowData - array of cell contents for a specific row
+   * @param {array} rowHeaders - array of table headers (column names)
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, rowData, rowHeaders) {
+    const hiddenHeaders = ['Cand ID', 'Session ID', 'Hide File', 'File Type'];
```

Comment:
As of PR #3955, hiddenHeaders should now be set and accessed as a state within this component.

Reply:
correct PR is https://github.com/aces/Loris/pull/3967 and was issued to major. changes might not be relevant to current PR which is going to minor